### ISSUE TITLE
 linter-pylama is not work with pylama package version  8.x.x

### ISSUE THREAD

**Issue by we684123** [AtomLinter/linter-pylama#124]
my linter-pylama is use "external"
and now latest version pylama package on pypi.org is 8.x.x
https://pypi.org/project/pylama/#history

linter-pylama can not work when pylama version is 8.x.x
latest available version is 7.7.1
`pip install pylama==7.7.1`

just notice to you (〜￣▽￣)〜

**Comment by wjandrea** [comment]
What's the problem exactly? I think I'm experiencing the same thing. `pylama: error: unrecognized arguments: -F`

<details>
  <summary>Full warning</summary>

  `usage: pylama [-h] [--version] [--verbose] [--options FILE] [--linters LINTERS] [--from-stdin] [--concurrent] [--format {pydocstyle,pycodestyle,pylint,parsable,json}] [--abspath] [--max-line-length MAX_LINE_LENGTH] [--select SELECT] [--ignore IGNORE] [--skip SKIP] [--sort SORT] [--report REPORT] [--hook] [--max-complexity MAX_COMPLEXITY] [--pydocstyle-convention {pep257,numpy,google}] [paths ...] pylama: error: unrecognized arguments: -F`
</details>

**Comment by wjandrea** [comment]
Turns out `-F`/`--force` was removed in Pylama 8.1.1, but with the two previous versions, 8.0.6 and 8.0.4, `linter-pylama` doesn't detect the syntax error here:

```py
something wrong
```

Installing Pylama 7.7.1 resolved the problem, thanks @we684123!

**Comment by we684123** [comment]
Github：「We are archiving Atom and all projects under the Atom organization for an official sunset on December 15, 2022.」
https://github.blog/2022-06-08-sunsetting-atom/
----
Thank you for developing the extension

**Comment by we684123** [comment]
and pyflakes need 2.4.0
```pip install pyflakes==2.4.0```
